src/index.html

```diff
@@ -15,8 +15,93 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="container">
+        <a
+          class="logo"
+          href="index.html"
+        >
+          <img
+            src="/src/images/logo.png"
+            alt="moyo"
```

Comment:
The `alt` attribute for the logo image should be more descriptive about the content. For example, `alt="Moyo Logo"`.